main/src/main/java/cgeo/geocaching/ui/SimpleItemListView.java

```diff
@@ -271,6 +274,33 @@ private void handleClick(final int adapterPos) {
         public void onBindViewHolder(@NonNull final ItemListViewHolder holder, final int position) {
             holder.fillData(getItem(position));
 
+            if (!model.getDisabledItems().isEmpty()) {
+                final boolean isEnabled = !model.getDisabledItems().contains(getItem(position).value);
+
+                ViewUtils.walkViewTree(holder.binding.itemViewAnchor, vi -> {
```

Comment:
The logic to make a view look "disabled/enabled" could probably be extracted into a Utility method (eg into ViewUtils) so it can be used by other places as well